packages/wallet/api/src/internal-types.js

```diff
@@ -52,8 +52,11 @@
  * @typedef {Object} Mapping
  * @property {(petname: Petname) => string} implode
  * @property {(str: string) => Petname} explode
- * @property {WeakStore<T, Petname>} valToPetname
+ * @property {LegacyWeakMap<T, Petname>} valToPetname
  * @property {WeakStore<T, string[][]>} valToPaths
+ *   TODO What about when useLegacyMap is true because contact have
```

Comment:
Is *contact* the word that is intended. There are several comments with the same word, but I can't make sense of them. Is this intended to be "content" or "contract", or am I just misunderstanding?

Reply:
This refers to the type `Contact` defined on line 29 above. The significance is that `valToPaths` might have `Contact` instances as keys, where these keys look like copyRecords---their passStyleOf is 'copyRecord`---but the lookup seems to need to use the identity of the records rather than a structural compare of their content. See https://github.com/Agoric/agoric-sdk/blob/c89503e54d56da50d9cda9d40fa21fe467da9e37/packages/wallet/api/src/lib-wallet.js#L116-L120

@michaelfig is this accurate? Should `valToPaths` be changed to a `LegacyWeakStore`?